src/esm_tests/test_utilities.py

```diff
@@ -261,6 +261,7 @@ def sort_dict(dict_to_sort):
         Dictionary sorted.
     """
     if isinstance(dict_to_sort, dict):
+        dict_to_sort = {str(key): value for key, value in dict_to_sort.items()}
```

Comment:
I'm not 100% sure this function will actually work for older Python version. Python dictionaries used to be unsorted by nature, only newer than 3.7 was order guaranteed to be kept same as insertion order (see here: https://realpython.com/sort-python-dictionary/#rediscovering-dictionary-order-in-python)

Not a problem with the code, just a "learn more about Python" moment 😉

Reply:
Thanks, good to know! :) Luckily this is only to make things more beautiful for the esm_tests output.